sdk/identity/identity/src/client/identityClient.ts

```diff
@@ -184,6 +190,36 @@ export class IdentityClient extends ServiceClient {
     }
   }
 
+  sendGetRequestAsync<T>(
+    url: string,
+    options?: NetworkRequestOptions
+  ): Promise<NetworkResponse<T>> {
+    const webResource = new WebResource(url, "GET", options?.body, {}, options?.headers);
+
+    return this.sendRequest(webResource).then((response) => {
+      return {
+        body: response.parsedBody as T,
+        headers: response.headers.rawHeaders(),
+        status: response.status
+      };
+    });
+  }
+
+  sendPostRequestAsync<T>(
```

Comment:
this can also be an async function. The `Async` suffix on these methods feels very .NET though, we don't follow that convention anywhere else.